xbmc/filesystem/ZipManager.h

```diff
@@ -32,12 +32,15 @@
 #define ECDREC_SIZE 22
 
 #include <memory.h>
+#include <regex>
 #include <string>
 #include <vector>
 #include <map>
 
 class CURL;
 
+static const std::regex PATH_TRAVERASL("(^|\\/|\\\\)\\.{2}($|\\/|\\\\)");
```

Comment:
spelling error 😄 
Also you can use a raw string `R"_((^|\/|\\)\.{2}($|\/|\\))_"` a bit messy but makes the regex a bit simpler to read